[2017-09-19 13:45:46 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'TYPE': u'WIDGET', u'OPERATION': u'INPUT', u'OBJECT': 330000018, u'DATA': 1, u'DATA_INP': [u'aaaa']}, u'MODE': None}}
[2017-09-19 13:45:48 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 13:45:58 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET_HTML', u'OBJECT': 330000018, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 13:45:59 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': u'<input clstag="h|keycount|2016|03a" autocomplete="off" id="key" accesskey="s" class="text" type="text">'}
[2017-09-19 13:46:18 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 13:46:19 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-19 16:56:43 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 16:57:23 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000003, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:01:32 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:01:32 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 67, in execute
    _status = self.__get_page(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 177, in __get_page
    self._root.get(_url)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 268, in get
    self.execute(Command.GET, {'url': url})
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: Failed to decode response from marionette


[2017-09-19 17:01:59 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:01:59 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 80, in start
    print 1
IOError: [Errno 32] Broken pipe

[2017-09-19 17:02:42 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:02:51 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000003, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:06:29 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:06:56 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000002, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:07:08 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:07:24 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 17:07:24 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-19 17:07:25 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:11:38 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:11:42 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:11:42 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 67, in execute
    _status = self.__get_page(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 177, in __get_page
    self._root.get(_url)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 268, in get
    self.execute(Command.GET, {'url': url})
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: Malformed URL: /Users/zhaojingping/Desktop/ttt.html is not a valid URL.


[2017-09-19 17:12:47 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:12:48 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:12:48 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 67, in execute
    _status = self.__get_page(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 177, in __get_page
    self._root.get(_url)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 268, in get
    self.execute(Command.GET, {'url': url})
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: Reached error page: about:neterror?e=fileNotFound&u=file%3A///zhaojingping/Desktop/ttt.html&c=UTF-8&f=regular&d=Firefox%20%E6%97%A0%E6%B3%95%E5%9C%A8%20/zhaojingping/Desktop/ttt.html%20%E6%89%BE%E5%88%B0%E8%AF%A5%E6%96%87%E4%BB%B6%E3%80%82


[2017-09-19 17:13:25 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:13:26 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:13:49 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 17:13:49 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-19 17:14:02 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:14:03 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:14:08 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:16:08 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:16:08 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:16:14 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:16:15 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:17:34 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:17:34 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:17:34 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 69, in execute
    _status = self.__action(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 118, in __action
    _node = WidgetBlock(self._root, p_para.cmd_object)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 60, in __init__
    self._get_widget()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 181, in _get_widget
    self._get_object(_detail)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 210, in _get_object
    self._widget = self.__get_child(_attr)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 219, in __get_child
    length = self._root.execute_script("arguments[0].ChildNodes.length;", self._widget)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 491, in execute_script
    'args': converted_args})['value']
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: TypeError: arguments[0].ChildNodes is undefined


[2017-09-19 17:17:42 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET_HTML', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:17:42 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:17:42 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 69, in execute
    _status = self.__action(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 118, in __action
    _node = WidgetBlock(self._root, p_para.cmd_object)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 60, in __init__
    self._get_widget()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 181, in _get_widget
    self._get_object(_detail)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 210, in _get_object
    self._widget = self.__get_child(_attr)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 219, in __get_child
    length = self._root.execute_script("arguments[0].ChildNodes.length;", self._widget)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 491, in execute_script
    'args': converted_args})['value']
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: TypeError: arguments[0].ChildNodes is undefined


[2017-09-19 17:22:52 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:23:09 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:23:14 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:23:24 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:23:24 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:23:24 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 69, in execute
    _status = self.__action(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 118, in __action
    _node = WidgetBlock(self._root, p_para.cmd_object)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 60, in __init__
    self._get_widget()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 181, in _get_widget
    self._get_object(_detail)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 210, in _get_object
    self._widget = self.__get_child(_attr)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 221, in __get_child
    return result[len(result)]
IndexError: list index out of range

[2017-09-19 17:24:13 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 17:24:13 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-19 17:24:17 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:24:29 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:24:34 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:24:40 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:24:41 driver.web.selenium] ERROR Driver run command failed!
[2017-09-19 17:24:41 driver.web.selenium] ERROR Traceback (most recent call last):
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/OrcDriver.py", line 120, in start
    _res = self._driver.web.execute(web_command)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 69, in execute
    _status = self.__action(p_cmd)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/WebDriver.py", line 118, in __action
    _node = WidgetBlock(self._root, p_para.cmd_object)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 60, in __init__
    self._get_widget()
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 181, in _get_widget
    self._get_object(_detail)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 210, in _get_object
    self._widget = self.__get_child(_attr)
  File "/Users/zhaojingping/PycharmProjects/AuxiTools/test/OrcTestToolsKit/OrcApi/OrcDriver/Web/Widget/OrcWidget.py", line 221, in __get_child
    return result[len(result)]
IndexError: list index out of range

[2017-09-19 17:25:33 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 17:25:33 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-19 17:25:42 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-19 17:25:57 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
[2017-09-19 17:26:02 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:26:14 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'EXISTS', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:26:15 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': True}
[2017-09-19 17:26:20 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET_TEXT', u'OBJECT': 330000028, u'TYPE': u'WIDGET'}, u'MODE': None}}
[2017-09-19 17:26:20 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run web command succeed', 'DATA': u'6'}
[2017-09-19 17:26:57 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-19 17:26:57 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-22 13:35:39 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-22 13:36:47 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'SYSTEM', u'CMD': {u'OPERATION': u'QUIT', u'DATA': None}}
[2017-09-22 13:36:47 driver.web.selenium] INFO Result is {'STATUS': True, 'MESSAGE': 'Run system command succeed.', 'DATA': None}
[2017-09-22 13:55:13 driver.web.selenium] INFO Start driver server on localhost:5001
[2017-09-22 13:55:30 driver.web.selenium] INFO Run command {u'SESSION': None, u'TYPE': u'DRIVER', u'CMD': {u'DTYPE': u'WEB', u'CMD': {u'OPERATION': u'GET', u'OBJECT': 300000006, u'TYPE': u'PAGE'}, u'MODE': None}}
